src/Microsoft.ML.Onnx/SaveOnnxCommand.cs

```diff
@@ -134,7 +137,51 @@ private void GetPipe(OnnxContextImpl ctx, IChannel ch, IDataView end, out IDataV
                 transform = (source = transform.Source) as IDataTransform;
             }
 
-            Host.AssertValue(source);
+            Contracts.AssertValue(source);
+        }
+
+        [BestFriend]
+        internal static ModelProto ConvertTransformListToOnnxModel(OnnxContextImpl ctx, IDataView inputData, IDataView outputData,
```

Comment:
Why `[BestFriend]`? I don't see that you're exposing this elsewhere.

Reply:
I will remove it. Thanks.

---
In reply to: [245215299](https://github.com/dotnet/machinelearning/pull/2013#discussion_r245215299) [](ancestors = 245215299)